src/parquet/util/compiler-util.h

```diff
@@ -31,8 +31,13 @@
 #undef UNLIKELY
 #endif
 
+#ifdef _MSC_VER
+#define LIKELY(expr) 0
```

Comment:
This should just "return" expr again, otherwise the statement would always be evaluated as false.